src/data_processing/wiki_to_netflix.py

```diff
@@ -0,0 +1,134 @@
+import requests
+import csv
+import os
+import math
+
+base_dir = os.path.join(os.path.dirname(__file__), '../data/')
+user_agent = 'Noisebridge MovieBot 0.0.1/Audiodude <audiodude@gmail.com>'
+
+# Reading netflix text file
+def read_netflix_txt(txt_file, test):
+    netflix_list = []
+    num_rows = 100 if test else math.inf
+
+    with open(txt_file, "r", encoding = "ISO-8859-1") as netflix_data:
+        for i, line in enumerate(netflix_data):
+            if i >= num_rows:
+                break
+            netflix_list.append(line.rstrip().split(',', 2))
+
+    return netflix_list
+
+# Writing netflix csv file
+def create_netflix_csv(csv_name, data_list):   
+    with open(csv_name, 'w') as netflix_csv:
+        csv.writer(netflix_csv).writerows(data_list)
+
+# Extracting movie info from Wiki data
+def wiki_feature_info(data, key):
+    if len(data['results']['bindings']) < 1 or key not in data['results']['bindings'][0]:
+        return 'NA'
+    if key == 'genreLabel':
+        return list({d['genreLabel']['value'] for d in data['results']['bindings'] if 'genreLabel' in d})
+    return data['results']['bindings'][0][key]['value'].split('/')[-1] 
+
+# Getting list of movie IDs, genre IDs, and director IDs from request
+def wiki_query(data_csv, user_agent):
+    wiki_movie_ids = []
+    wiki_genres = []
+    wiki_directors = []
+        
+    for row in data_csv:
+        if row[1] == "NULL":
+            continue
+
+        SPARQL = '''
```

Comment:
Maybe move the triple quoted string up to the top level and leave it as the constant `SPARQL` then do something like:

```
query = SPARQL % {'Title': row[2], 'Year': int(row[1])}
```

That would make this function a lot easier to read.

Reply:
I did that and refactored the query construction to be in its own function. Should be a bit more solid.

Reply:
I dont' see those changes, did you push them?

Reply:
Okay I see it now thanks!